package.json

```diff
@@ -34,11 +33,10 @@
     "mocha": "^5.1.0",
     "pixelmatch": "^4.0.2",
     "polymer-cli": "^1.7.1",
-    "polyserve": "^0.27.0",
```

Comment:
I think `polyserve` is needed for screenshot testing.

Reply:
yes it definitely is!